main.js

```diff
@@ -160,6 +160,8 @@ define(function (require, exports, module) {
                     if (key) {
                         base = _getBaseKey(i);
                         command = CommandManager.get(key.commandID);
+                        if(command === undefined)
+                            continue;
```

Comment:
This should check for `!command` to also cover null case. Also, I prefer braces and space between "if" and "(":

```
    if (!command) {
        continue;
    }
```

Thanks.

Reply:
Done and done.
Is there a proper way to indicate that I've applied the requested changes on gitHub? I don't see an option for that